make.go

```diff
@@ -331,9 +335,9 @@ func getDebianEmail() string {
 	if email := strings.TrimSpace(os.Getenv("DEBEMAIL")); email != "" {
 		return email
 	}
-	if mailname, err := ioutil.ReadFile("/etc/mailname"); err == nil {
-		if u, err := user.Current(); err == nil {
-			return u.Name + "@" + strings.TrimSpace(string(mailname))
+	if mailname, err := ioutil.ReadFile("/etc/mailname"); err == nil && strings.Contains(string(mailname), ".") {
```

Comment:
This is getting a bit long. Can you change it to the following please?

```go
mailname, err := ioutil.ReadFile("/etc/mailname")
if err == nil && strings.Contains(string(mailname), ".") {
  // …
```

Also, can you please add a comment describing why you check for a dot being in the mailname? I know it’s to exclude default settings like “debian”, but I think it should be documented in the code :).